src/metric_config.cpp

```diff
@@ -107,7 +107,12 @@ void MetricConfig::setDefaultMetricsTo(bool enabled) {
     }
 }
 
-Status MetricConfig::loadFromCLIString(bool isEnabled, const std::string& metricsList) {
+Status MetricConfig::loadFromCLIString(bool isEnabled, const std::string& metricsList, uint64_t restPort) {
+    if (isEnabled && restPort == 0) {
```

Comment:
I have tested this manually and this does not work. Running config.json with monitoring enabled and missing `--rest_port` still starts OVMS normally.
What's the point of adding this parameter here instead of validating it in ovms::Config class?